src/rustdoc.md

```diff
@@ -60,6 +60,17 @@ does is call the `main()` that's in this crate's `lib.rs`, though.)
 * Use `./x test tests/rustdoc*` to run the tests using a stage1
   rustdoc.
   * See [Rustdoc internals] for more information about tests.
+* Use `./x test tidy --extra-checks=js` to run ESLint/TSC checks on rustdoc's frontend code.
```

Comment:
this command will also run `es-check`.  also note that `./x test tidy` will (by default) automatically run these checks if there are changes to javascript source files.